scene-lib/src/scenelib/annotations/el/AScene.java

```diff
@@ -141,6 +142,16 @@ public int hashCode() {
         return classes.hashCode() + packages.hashCode();
     }
 
+    /**
+     * Fetch the classes in this scene, represented as AClass objects.
+     *
+     * @return an immutable map from binary names to AClass objects
+     */
+    @SuppressWarnings("signature:return.type.incompatible") // unannotated ImmutableMap??
```

Comment:
the `classes` field isn't annotated, but should be.

Reply:
We could remove this warning suppression if we annotated it, but that would introduce more work (and annotations) elsewhere. I'm okay with leaving this suppression if you don't want to make more changes in scene-lib, but we should at least update this comment to say why the warning is needed (`classes` is unannotated).

Reply:
scene-lib is not verified, so I should not have copied this warning suppression from the Checker Framework.